db/migrate/20170609195100_remove_codelocation_status_default.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveCodelocationStatusDefault < ActiveRecord::Migration
+  def self.up
+    change_column_default :code_locations, :status, nil
```

Comment:
@msk999, please do let us the reason for changing **code_locations#status** to nil.